js/pages/incidence-rates/ir-manager.js

```diff
@@ -246,7 +243,16 @@ define([
 				this.pollForInfo();
 				this.pollTimeout = PollService.add(() => this.pollForInfo({ silently: true }), 10000);
 			});
-		};
+		}
+
+		onRouterParamsChanged(params = {}) {
+			const { analysisId } = params;
+			if (!analysisId) {
```

Comment:
Why `parseInt` isn't used here? If I'm not mistaken, `analysisId` in such case equals to `"0"` (string)